### PR TITLE
OADP-773:Velero version by OADP and OCP version

### PR DESCRIPTION
**Pull request by RichardHoch**
OADP 1.1.0, OCP 4.6+

Resolves https://issues.redhat.com/browse/OADP-773 by changing the reference to Velero version 1.7 in https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html-single/backup_and_restore/index#application-backup-restore-operations-overview and by modfying the table in https://access.redhat.com/documentation/en-us/openshift_container_platform/4.11/html/backup_and_restore/application-backup-and-restore#velero-obtaining-by-downloading_oadp-troubleshooting

Previews:
1.  https://53184--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/index.html#application-backup-restore-operations-overview [reference to Velero version changed to reference to "Downloading the Velero CLI tool] 

2.  https://53184--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/troubleshooting.html#velero-obtaining-by-downloading_oadp-troubleshooting [Table 4.2]

### CODE REVIEW

**Comment by RichardHoch** [comment]
@kaovilai Please review this PR.

**Review by kaovilai — APPROVED** [review]
/lgtm

**Comment by RichardHoch** [comment]
@mperetzred -- please review.

File: modules/velero-obtaining-by-downloading.adoc
Comment (mburke5678) [inline modules/velero-obtaining-by-downloading.adoc:26]: The {OCP} is not rendering in the preview. Should it? I think you need an additional `include::_attributes/...` line, either to `attributes-microshift.adoc` or `attributes-openshift-dedicated` it appears.
I don't see any other instances of {OCP} (OCP) elsewhere in the docs. 

On second thought, I think this should be {product-title}, as this module [appears in the OKD docs](https://docs.okd.io/latest/backup_and_restore/application_backup_and_restore/troubleshooting.html#velero-obtaining-by-downloading_oadp-troubleshooting).

File: modules/velero-obtaining-by-downloading.adoc
Comment (mburke5678) [inline modules/velero-obtaining-by-downloading.adoc:28]: Same issue here and below with the OCP/OKD. I wonder if `.OADP-Velero-{product-title} version` will work here.

File: modules/velero-obtaining-by-downloading.adoc
Comment (mburke5678) [inline modules/velero-obtaining-by-downloading.adoc:31]: s/OCP/{product-title} ??

**Comment by mburke5678** [comment]
@RichardHoch A few nits. Othwerwise LGTM

File: modules/velero-obtaining-by-downloading.adoc
Comment (RichardHoch) [reply modules/velero-obtaining-by-downloading.adoc:26]: @mburke5678 From what i understand from https://issues.redhat.com/browse/OADP-773, we want this PR be part of the  OCP documentation as usual -- did I build the file structure incorrectly? From your comments it sounds like it will appear only in OKD docs.

File: modules/velero-obtaining-by-downloading.adoc
Comment (kaovilai) [inline modules/velero-obtaining-by-downloading.adoc:36]: Added info for 1.1.1 and 1.1.2
```suggestion
|1.1.0 |link:https://{velero-domain}/docs/v{velero-version}/[{velero-version}] |4.9 and later
|1.1.1 |link:https://{velero-domain}/docs/v{velero-version}/[{velero-version}] |4.9 and later
|1.1.2 |link:https://{velero-domain}/docs/v{velero-version}/[{velero-version}] |4.9 and later
```

Comment (RichardHoch) [reply modules/velero-obtaining-by-downloading.adoc:36]: Will do.

**Comment by RichardHoch** [comment]
@mburke5678 I made the changes you requested. 

Please merge this PR if it passes review (no need to wait for the next OADP release).

File: modules/velero-obtaining-by-downloading.adoc
Comment (adellape) [inline modules/velero-obtaining-by-downloading.adoc:35]: The URLs here are invalid. Guessing maybe missing some attributes like the others below are using?

**Review by adellape — COMMENTED** [review]
@RichardHoch Please see 1 new comment and also squash down to 1 commit. Removing the merge-review labels for now; please resubmit to the merge queue when it's ready. Thank you!

File: modules/velero-obtaining-by-downloading.adoc
Comment (RichardHoch) [reply modules/velero-obtaining-by-downloading.adoc]: @adellape Fixed -- thanks!

**Comment by RichardHoch** [comment]
If good to merge, please merge today. Thanks!

**Comment by jeana-redhat** [comment]
@RichardHoch hey, sorry this took a while to cherrypick! Seems the bot was acting up. I will follow through to merge 4.11-4.13, but you will need to open manual cherrypicks for 4.9 + 4.10

**Comment by RichardHoch** [comment]
@jeana-redhat Will do

**Pull request merged**
